adapters/repos/db/index_queue.go

```diff
@@ -631,3 +751,100 @@ type chunk struct {
 	data     []vectorDescriptor
 	elem     *list.Element
 }
+
+type minIndexedCursor struct {
+	sync.Mutex
+	id uint64
+	f  *os.File
+
+	flushCh chan struct{}
+}
+
+func newMinIndexedCursor(shardID string, rootPath string) (*minIndexedCursor, error) {
+	fileName := fmt.Sprintf("%s/%s.minindexed", rootPath, shardID)
```

Comment:
Wonder if we should be creating a new subdirectory for this and future vector on disk formats. @etiennedi do you have any thoughts here?

Reply:
With @abdelr we are going to move this to the commit log, because as it is now the data is lost during a migration.